presto-plugin-toolkit/src/test/java/com/facebook/presto/plugin/base/security/TestFileBasedAccessControl.java

```diff
@@ -18,13 +18,14 @@
 import com.facebook.presto.spi.connector.ConnectorTransactionHandle;
```

Comment:
I think that commit message could be a bit better. `Cleanup...` is very generic.